amundsen_application/static/js/components/ColumnList/index.tsx

```diff
@@ -274,6 +288,18 @@ const ColumnList: React.FC<ColumnListProps> = ({
     ];
   }
 
+  if (hasColumnBadges) {
+    formattedColumns = [
+      ...formattedColumns,
+      {
+        title: 'Badges',
+        field: 'badges',
+        horAlign: TextAlignmentValues.left,
+        component: (values) => (values ? <BadgeList badges={values} /> : null),
```

Comment:
If we default to [], then this is going to be called always, so we won't need to use the ternary.
In this ocassion the BadgeList is handling the empty list case.